extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -49,6 +49,25 @@
    transition: transform 233ms cubic-bezier(0,0,.21,1);
  }
 
+  header[toolbar] nav[toolbar] {
```

Comment:
Decided to add a class to the target element, rather than forcing an attribute on the target